tests/openvino/test_modeling.py

```diff
@@ -1878,14 +1891,15 @@ def test_compare_to_transformers(self, model_arch):
         gen_config = GenerationConfig(
             max_new_tokens=30,
             min_new_tokens=30,
-            num_beams=3,
             do_sample=False,
             eos_token_id=None,
         )
         set_seed(SEED)
         ov_outputs = ov_model.generate(**inputs, generation_config=gen_config)
         set_seed(SEED)
         transformers_outputs = transformers_model.generate(**inputs, generation_config=gen_config)
+        if model_arch == "minicpmv":
+            ov_outputs = ov_outputs[:, inputs["input_ids"].shape[1] :]
```

Comment:
why ?

Reply:
original pytorch model does not returns input tokens as outputs, while optimum behavior aligned with standard transformers API do that